src/main/java/org/jenkinsci/test/acceptance/junit/JenkinsAcceptanceTestRule.java

```diff
@@ -56,6 +70,14 @@ public void evaluate() throws Throwable {
 
                 injector.injectMembers(this);
 
+                if (enableInstallWizard) {
```

Comment:
Note that in case of pooled controller it can be started already so this will have no effect.